src/main/java/model/process/analysis/operations/comparisons/LagSequentialAnalysis.java

```diff
@@ -1,21 +1,233 @@
 package model.process.analysis.operations.comparisons;
 
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+
+import model.data.DataRow;
+import model.data.DataTable;
+import model.data.DataTableBuilder;
+import model.data.Table;
+import model.data.describer.DataDescriber;
+import model.data.value.DateTimeValue;
+import model.exceptions.InputMismatchException;
 import model.process.analysis.operations.Event;
 
 /**
- * This class will determine a relation between events.
- * Relation will be shown by chronologically sorting events.
+ * This class will determine a relation between events. Relation will be shown
+ * by chronologically sorting events.
  * 
  * @author Louis Gosschalk 26-05-2015
  */
 public class LagSequentialAnalysis {
-	
+
+	private DataTable tableA;
+	private DataTable tableB;
+	private DataTable result;
+
+	private DataDescriber<DateTimeValue> dateA;
+	private DataDescriber<DateTimeValue> dateB;
+
+	private int positionA;
+	private int positionB;
+
+	private DateTimeValue compareA;
+	private DateTimeValue compareB;
```

Comment:
Iterators heb ik hier geprobeerd maar geven te veel problemen. jammer genoeg weet ik niet meer de exacte reden maar in de bezigheid bewees dit de beste oplossing